source/manuals/programming-languages/html.html.md.erb

```diff
@@ -1,85 +1,263 @@
+<br />
+<img src="foo.jpg" alt="example" />
+<p>This is a paragraph.</p>
 
-The following landmark roles don't map to HTML5 elements, but are
-still good to use:
+<!-- Bad -->
+<br>
+<img src="foo.jpg" alt="example">
+<p>Notice no closing tag.
+<img src="bar.jpg" alt="another example"></img>
+```
 
-* `role="search"`
-* `role="application"`
+### Attributes
 
-[Demo of how landmark roles help screen reader users](http://tink.co.uk/2011/07/how-do-aria-landmark-roles-help-screen-reader-users/)
+Always quote attribute values. Quotes should be double-quoted (`"`) rather
+than single-quoted (`'`) unless you're rendering HTML through template strings
+(e.g. in JavaScript), in which case either single or double quotes is fine.
 
-## Main content
+Some attributes don't require values. In these cases, omit them.
 
-The `<main>` tag should be used to identify the main content of your page. Support for it is still marginal so ensure to add the `role="main"` attribute.
+```html
+<!-- Bad -->
+<img height=30 width=30 src="foo.png" alt="logo" />
+<input type="checkbox" name="vehicle3" value="Boat" checked="checked" />
 
-## Sectioned content
+<!-- Good -->
+<img height="30" width="30" src="foo.png" alt="logo" />
+<input type="checkbox" name="vehicle3" value="Boat" checked />
+```
 
-When using the `<section>` tag, add an `aria-labelledby` attribute linking to the id of the heading that identifies that section.
+## Document structure
 
-Example:
+We use HTML5 to structure our page semantically. We also use ARIA roles to help
+older assistive technologies map the sections correctly.
+
+Léonie Watson explains [why using ARIA landmark roles in this way delivers the
+best experience for screen reader users](http://tink.uk/screen-readers-aria-roles-html5-support/).
+
+### Header
+
+`<header role="banner">` should be used to contain your home link, branding, search,
+and any global navigation you may have.
 
-    <section aria-labelledby="advice">
-      <h1 id="advice">Help and advice</h1>
+### Main content
 
-## Navigational elements
+`<main role="main">` should be used to identify the main content of your page.
 
-When using the `<nav>` tag, add the ARIA landmark `role="navigation"`.
+A "Skip to main content" link should be added to help screen reader and keyboard
+users skip past your general site navigation and get straight to the content.
+You can use the [GOV.UK skip link component](https://design-system.service.gov.uk/components/skip-link/)
+for this.
 
-The detail of using the `<nav>` tag is covered in this [blog post](https://insidegovuk.blog.gov.uk/2013/07/03/rethinking-navigation/).
+### Sectioned content
 
-### Emphasis
+Use the `<section>` tag to represent standalone sections where a more specific
+semantic element would not be appropriate.
 
-If you want to give the current location emphasis within a navigational list, use a `<strong>` tag.
+You should add an `aria-labelledby` attribute linking to the id of the heading
+that identifies that section.
```

Comment:
I thought about that, and the previous version did have an Accessibility section, but that seems to be a throwback to the days where accessibility is tagged onto the end of a project rather than considered everywhere.

There's usually some aspect of a11y to consider at every point in our HTML, so having it in a separate section feels reductive to me 🤔

Agreed that we should talk this through at the frontend meetup.

Reply:
In this case, the sectioned elements similar to a `div` if it's not given a correct label. With it's label it'll become a useful landmark. So it works well where you have proposed @ChrisBAshton